.changeset/afraid-squids-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+Fix e2e by stoppin,g the carousel in onboarding
```

Comment:
There's a typo here 😉 
Also I'm not sure a changelog is necessary for this kind of PR